docs/devcontainers/tutorial.md

```diff
@@ -58,7 +58,7 @@ With the Dev Containers extension installed, you will see a new Status bar item
 
 The Remote Status bar item can quickly show you in which context VS Code is running (local or remote) and clicking on the item will bring up the Dev Containers commands.
 
-![Dev Containers commands](images/tutorial/dev-containers-commands.png)
+![Dev Containers commands](images/tutorial/dev-containers-commands-simple.png)
```

Comment:
There are actually some unshipped updates to the Codespaces extension (mostly appending `...` to the Connect/Create/Delete commands). VSIX here https://github.com/github/codespaces-vscode/suites/11140676895/artifacts/568247613 or if you don't mind, I can push directly to this branch with an updated screenshot.

Reply:
Yes please feel free to push directly to this branch, thank you!

Reply:
It looks like you are using the stable dark theme rather than the experimental one, because the focus color is different. Should I add screenshots using the stable dark theme until we make the experimental dark theme the default?

Reply:
I don't have a strong preference either way. For a bit of future-proofing, I'd be fine with showing the experimental one, if the parts that are shown in the screenshot aren't likely to change much. cc @daviddossett @gregvanl for opinions

Reply:
Oops--I went ahead with the stable dark theme to avoid confusion and since I'm unsure whether we plan to keep updating the experimental theme.

Reply:
The current stable dark theme would be my preference